flask_admin/static/admin/js/form-1.0.0.js

```diff
@@ -155,7 +155,7 @@
         }
 
         // set up tiles
-        L.tileLayer('http://{s}.tiles.mapbox.com/v3/'+MAPBOX_MAP_ID+'/{z}/{x}/{y}.png', {
```

Comment:
The v3 API is still accessible for maps created before mid March, but for anything newer, only v4 is supported. Earlier maps will work just fine with the v4 API, people will have to add MAPBOX_ACCESS_TOKEN if they upgrade flask-admin.